CODEOWNERS

```diff
@@ -222,6 +223,7 @@
 /include/drivers/display.h                @vanwinkeljan
 /include/drivers/espi.h                   @albertofloyd @franciscomunoz @scottwcpg
 /include/drivers/bluetooth/               @joerchan @jhedberg @Vudentz
+/include/drivers/flash.h                  @nvlsianpu
```

Comment:
Note that this line overrides the /include/ line above, so you'd be left as the only automatic review for this /include/drivers/flash.h file @MaureenHelm @galak @nashif are you also ok with it?

Reply:
@aescolar This was not my intention. I didn't know that assignments are resolved like that. Will add all of the here back.

Reply:
(yep, it is pretty un-intuitive..)